src/interfaces/simulation.spec.ts

```diff
@@ -229,6 +248,45 @@ describe("Simulation interface", () => {
         expect(error).toHaveProperty("error_code", PriceFetchingError.FETCHING_PRICE_ORACLE);
       }
     });
+
+    describe("when it does not have the partner service", () => {
+      it("should directly call the vault contract", async () => {
+        tokenMock.mockReturnValueOnce(Promise.resolve("0x001"));
+        executeSimulationWithReSimulationOnFailureSpy.mockImplementationOnce(fn => fn());
+
+        await simulationInterface.deposit("0x000", "0x001", "1", "0x0000", { slippage: 1 });
+        expect(executeSimulationWithReSimulationOnFailureSpy).toHaveBeenCalledTimes(1);
+        expect(partnerEncodeDepositMock).toHaveBeenCalledTimes(0);
+        expect(simulateVaultInteractionSpy).toHaveBeenCalledTimes(1);
+        expect(simulateVaultInteractionSpy).toHaveBeenCalledWith("0x000", "0x0000", "", "0x0000", {
+          forkId: undefined,
+          root: undefined,
+          save: undefined,
+          slippage: 1
+        });
+      });
+    });
+
+    describe("when it has the partner service", () => {
+      it("should directly call the vault contract", async () => {
```

Comment:
wrong description?

Reply:
This was fixed with https://github.com/yearn/yearn-sdk/pull/224/commits/f3c8b635a06a9128e9802b2f4f2dacb8a859c010#diff-3b83facde1670849a263b00b34c55247d9070641528b6579edd9ed4e9445f3a5R253